src/array.js

```diff
@@ -115,21 +114,21 @@ inherits(ArraySchema, MixedSchema, {
     return next;
   },
 
-  required(message = mixed.required) {
+  required(message = null) {
```

Comment:
i don't think the default to `null` is necesary is it?

Reply:
right, just for clarity and consistency